src/search/abstract_task.cc

```diff
@@ -14,6 +14,21 @@ ostream &operator<<(ostream &os, const FactPair &fact_pair) {
     return os;
 }
 
```

Comment:
it might be good to add a comment that this shouldn't be called repeatedly in efficiency critical parts of the code since it is a static property that can just be computed once

Reply:
Does this refer to `has_conditional_effects`? I could add a comment, but this has nothing to do with this issue.

Reply:
Yes it does. The diff view shows me that this function has been added.

Reply:
You are right, my bad. I thought I got this from somewhere else and only exposed it, but I can't find it anymore. I updated the comment in the header file.

Reply:
After reading my previous comment I realize that it sounds very harsh, but it wasn't intended that way. My "yes it does" was supposed to answer your question, I didn't mean to insist it has something to do with the issue!